Headers/AppKit/NSSpeechSynthesizer.h

```diff
@@ -151,8 +151,13 @@ extern NSString *NSSpeechDictionaryEntryPhonemes;
 - (NSString *) phonemesFromText: (NSString *)text;
 @end
 
-// Speech synthesizer delegate informal protocol...
-@interface NSObject (NSSpeechSynthesizerDelegate)
+@protocol NSSpeechSynthesizerDelegate <NSObject>
+#if GS_PROTOCOLS_HAVE_OPTIONAL
+@optional
+#else
+@end
+@interface NSObject (NSURLConnectionDelegate)
```

Comment:
Thank you. Has been fixed.

(I used `NSURLConnectionDelegate` mistakenly because it was my reference for this change.)